src/main/kotlin/minesweeper/view/BoardView.kt

```diff
@@ -0,0 +1,26 @@
+package minesweeper.view
+
+import minesweeper.domain.Board
+import minesweeper.domain.Cell
+import minesweeper.domain.Mine
+import minesweeper.domain.Opened
+
+class BoardView(private val board: Board) {
+    fun print() {
+        board.cells.map { getCharacter(it) }
+            .chunked(board.width)
+            .forEach { println(it.joinToString(" ")) }
```

Comment:
객체 내부속성을 외부에서 직접 사용하지 말고 객체에게 메시지를 보내서 일을 시켜보면 어떨까요?
```suggestion
        board.rows()
            .forEach { ...
```
[Tell, don't ask](https://www.google.com/search?client=safari&rls=en&q=Tell%2C+don%27t+ask&ie=UTF-8&oe=UTF-8) 를 지키다보면 좀 더 객체지향적으로 바뀌는것 같아요.